Model/ApiPayloadRequest.php

```diff
@@ -63,6 +63,15 @@ public function __construct($request, Transport $transport, TokenHelper $tokenHe
      */
     public function send()
     {
+        // If it's in test mode, add test_values to TokenHelper.
+        if ($this->test && !is_null($this->request) && !is_null($this->request->body)) {
```

Comment:
Screw !,

Why not: 
 is_null($this->request)  === false?

Much more readable.